docusaurus.config.js

```diff
@@ -12,7 +12,7 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 const config = {
   title: 'LoopBack',
   tagline: 'Dinosaurs are cool',
-  url: 'https://loopbackio.github.io',
+  url: 'https://blog.loopback.io',
```

Comment:
Non-blocker; Are we planning to use the `blog` subdomain, or still going forward with <https://loopback.io/blog>? I'm currently writing a shell script for to vendor this repository into <https://github.com/loopbackio/loopback.io>. Still working out the kinks, but will open a PR there once it's ready.

Reply:
@achrinza, I cannot find a way to have GitHub pages from 2 repos using the same domain, so subdomain `blog.loopback.io` is being used. If you find a way to do it, I can update it as well.

Reply:
This has been resolved in https://github.com/loopbackio/loopback-blog/pull/9